lib/cocoapods/generator/app_target_helper.rb

```diff
@@ -100,7 +100,7 @@ def self.add_xctest_search_paths(target)
       #
       def self.add_swift_version(target, swift_version)
         target.build_configurations.each do |configuration|
-          configuration.build_settings['SWIFT_VERSION'] = swift_version
+          configuration.build_settings['SWIFT_VERSION'] = !swift_version.nil? ? swift_version : Validator::DEFAULT_SWIFT_VERSION
```

Comment:
needs a test

Reply:
Nit pick here, rather than checking for nil in each loop, could the desired value be extracted to before the loop? Something like

```
version_to_set = swift_version || Validator::DEFAULT_SWIFT_VERSION
target.build_configurations.each do |configuration|
    configuration.build_settings['SWIFT_VERSION'] = version_to_set
end
```

Reply:
sure! let me do that